redash/query_runner/mysql.py

```diff
@@ -46,9 +46,25 @@ def configuration_schema(cls):
                     'type': 'string',
                     'title': 'Database name'
                 },
-                "port": {
-                    "type": "number"
+                'port': {
+                    'type': 'number'
                 },
+                'use_ssl': {
+                    'type': 'boolean',
+                    'title': 'Use SSL'
+                },
+                'ssl_cacert': {
+                    'type': 'string',
+                    'title': 'CA certificate to verify peer against (SSL)'
```

Comment:
I want to keep re:dash instances stateless as much as possible. This is why the BigQuery and Google Spreadsheets query runners store the credentials file in the DB. I prefer we do the same here.

If MySQL client only accepts file paths for SSL certificates, we can write those to a temp file before connecting. If this becomes too much of an overhead, we can "cache" the file (write it always to the same path, so it's not needed to be written more than once).